_data/docs_v5.yml

```diff
@@ -9,13 +9,15 @@
 
 - title: Configuration
   docs:
+    - format
     - plugins
     - target
     - command
     - telemetry
     - interfaces
     - protocols
     - table
+    - telemetry_screens
```

Comment:
Screens are just configuration files so I think it makes sense to move it here ... kept the file name so any links are still ok